test/integration/node-specific/client_close.test.ts

```diff
@@ -265,7 +270,6 @@ describe.skip('MongoClient.close() Integration', () => {
 
                     while (heartbeatOccurredSet.size < servers.size) {
                       const ev = await once(client, 'serverHeartbeatSucceeded');
```

Comment:
Intentional to remove the logs from these tests?

Reply:
This predates my changes, they aren't used for any asserting so maybe they were debugging, they could be added back if these fail, but they won't be visible in CI so you'd need to re-run it anyway to grab the info printed here.